Dreamscapes/Ldap/Core/Ldap.php

```diff
@@ -752,11 +757,10 @@ public function __call($method, $args)
         return call_user_func_array([$this, 'ldapSearch'], $args);
     }
 
-
```

Comment:
I always separate methods' visibility change with two empty lines. E.g. first there are public methods, <2 lines>, protected methods, <2 lines>, private methods. It helps me visually distinguish these groups.

Please revert this change.